src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="is-active nav__link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
```

Comment:
```suggestion
          <li class="nav_item">
            <a
              href="#"
              class="is-active nav__link"
            >
              Apple
            </a>
          </li>
          
          <li class="nav_item">
```

add an empty line between multiple sibling blocks of html

Reply:
Not fixed 
<img width="425" alt="Screenshot 2024-04-12 at 15 37 20" src="https://github.com/mate-academy/layout_moyo-header/assets/135232001/379760fc-4c81-4fca-8093-01e1c0025b1b">